### PR TITLE
Static content

### PR DESCRIPTION
**Pull request by keveri**
Add static resource folder and serve it with http4s.

### CODE REVIEW

**Comment by keveri** [comment]
related to issue #8

**Comment by keveri** [comment]
based on this example: https://github.com/http4s/http4s_demo/blob/57a9179165fb1ffb43dcde160e5ef85d74f178a6/src/main/scala/org/http4s/example/Routes.scala

File: src/main/scala/neljas/Http.scala [hunk 1]
```diff
@@ -5,12 +5,18 @@ import org.http4s.MediaType._
...
+import org.http4s.server.staticcontent.ResourceService.Config
 
 import neljas.pdf.PDF
 
 object Http extends TwirlInstances {
 
+  private val static = cachedResource(Config("/static", "/static"))
+
   val route = HttpService {
+    case r @ GET -> _ if r.pathInfo.startsWith("/static") => static(r)
```
Comment (jkpl) [inline src/main/scala/neljas/Http.scala]: Did you try? `case req @ GET -> "static" /: path =>`

**Comment by jkpl** [comment]
Looks good!

**Pull request merged**